requirements.txt

```diff
@@ -1,6 +1,10 @@
 inputs==0.1
 wxPython==4.0.0a1
-matplotlib==1.5.2
-numpy==1.10.1
-scikit-image==0.12.3
+matplotlib==2.0.0
+numpy==1.12.1
+scikit-image==0.13.0
```

Comment:
are these updates required? These were the latest on my system so I'm assuming these are the default versions in a new-ish linux system. The change is probably fine but I'd also rather not do it unless there is a reason to.